Weekly CI で muon-master job が失敗した (2025-03)

Weekly CI で muon-master job が失敗した

<!-- SPDX-License-Identifier: FSFAP -->
<!--
バグの報告ありがとうございます！
以下の項目は修正の手掛かり・助けになりますので記入をお願いいたします。
不具合なのかはっきりしない動作や挙動はDiscussionsで質問してみてください。
-->

**バグの説明**
<!-- 何が起こったか、エラーメッセージがあればそれも書く -->

GitHub Actions の Weekly CI で muon-master job が失敗したと通知が届きました。

https://github.com/JDimproved/JDim/actions/runs/13889399067/job/38858735256

muon-master job の raw logs (抜粋):
```
2025-03-17T01:11:26.3321889Z ##[group]Run cd muon-src
2025-03-17T01:11:26.3322166Z [36;1mcd muon-src[0m
2025-03-17T01:11:26.3322659Z [36;1m./stage1/muon-bootstrap setup -Ddocs=disabled -Dlibarchive=disabled -Dlibcurl=disabled -Dlibpkgconf=enabled ../muon-build[0m
2025-03-17T01:11:26.3323232Z [36;1m./stage1/muon-bootstrap samu -C ../muon-build[0m
2025-03-17T01:11:26.3323497Z [36;1mcd ..[0m
2025-03-17T01:11:26.3369309Z shell: /usr/bin/bash -e {0}
2025-03-17T01:11:26.3369520Z env:
2025-03-17T01:11:26.3369677Z   CC: gcc-13
2025-03-17T01:11:26.3369842Z   CXX: g++-13
2025-03-17T01:11:26.3370000Z ##[endgroup]
2025-03-17T01:11:26.3458861Z err invalid option: 'docs=disabled'
2025-03-17T01:11:26.3472871Z ##[error]Process completed with exit code 1.```
```

**再現の方法**
<!-- バグの動作を再現する手順を書く -->
```sh
git clone --depth 1 https://git.sr.ht/~lattis/muon muon-src
cd muon-src
./bootstrap.sh stage1
./stage1/muon-bootstrap setup -Ddocs=disabled ./stage2
```

**やりたかったこと・期待する結果**
<!-- 本来の望ましい動作を書く -->
job を実行して成功すること。

**スクリーンショット**
<!-- 表示の問題はスクリーンショットがあれば分かりやすい -->
なし

**動作環境**
<!-- クリップボードへのコピーを利用する -->

JDim:
```
[バージョン] JDim 0.14.0-alpha20250315(git:be72d9f257)
[ディストリ ] Ubuntu 24.10 (x86_64)
[パッケージ] バイナリ/ソース( <配布元> )
[ DE／WM ] KDE (Wayland)
[　gtkmm 　] 3.24.9
[　glibmm 　] 2.66.7
[　TLS lib　] GnuTLS 3.8.6
[オプション ] '--with-alsa''--with-pangolayout'
[ そ の 他 ] 
```

muon:
```
commit 7882bc6805483334dbc1196aff0d1671622f4aa3 (HEAD -> master, origin/master, origin/HEAD)
Author: Stone Tickle <lattis@mochiro.moe>
Date:   2025-03-16 09:00:57 -0400

    fix some var source locations

    also make goto definition not crash if the source location is unknown
```

**追加の情報**
<!-- ユーザーエージェントなどが手掛かりとなる情報 -->

muon リポジトリに対して git bisect で原因のコミットを探索した結果です。
```
9da9267b8564e81d16aa7bd732198f0c4549e5d6 is the first bad commit
commit 9da9267b8564e81d16aa7bd732198f0c4549e5d6
Author: Stone Tickle <lattis@mochiro.moe>
Date:   2025-03-02 22:14:32 -0500

    always build docs

 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
bisect found first bad commit
```

bisect で検出したコミットを見ても原因がよく分かりませんでした。
そこで、エラーメッセージに含まれている `docs=disabled` をヒントにコミットログを検索し、関係がありそうなコミットを見つけました。

```
commit f94e3c889238d0642cfaebdf6905f2ad88db1065
Author: Stone Tickle <lattis@mochiro.moe>
Date:   2025-03-14 06:41:20 -0400

    clean up docs and tests related options

    - separate options to control meson-docs and meson-tests
    - decouple docs and website
        docs => man-pages
    - change website to a 'feature'
```

--- a/.github/workflows/weekly.yml
+++ b/.github/workflows/weekly.yml
@@ -81,7 +81,7 @@ jobs:
       - name: build muon stage2
         run: |
           cd muon-src
-          ./stage1/muon-bootstrap setup -Ddocs=disabled -Dlibarchive=disabled -Dlibcurl=disabled -Dlibpkgconf=enabled ../muon-build
+          ./stage1/muon-bootstrap setup -Dman-pages=disabled -Dmeson-docs=disabled -Dmeson-tests=disabled -Dlibarchive=disabled -Dlibcurl=disabled -Dlibpkgconf=enabled ../muon-build
           ./stage1/muon-bootstrap samu -C ../muon-build
           cd ..
       - run: ./muon-build/muon version

